dyndxt_loader/loader.py

```diff
@@ -0,0 +1,263 @@
+"""Injects and manages the Dynamic DXT loader."""
+import copy
+import os
+import struct
+
+from xboxpy.interface import if_xbdm
+from .export_info import ExportInfo
+from .py_dll_loader import DLLLoader
+from . import xbdm_exports
+from . import xboxkrnl_exports
+
+
+_XBDM_HOOK_COMMAND_FMT = "resume thread=0x%X"
```

Comment:
Are there any issues with using the same XBDM command as the xboxpy hack here? If someone attempts to use the xbdm hack after this is loaded, what's the behavior?